src/components/Home/Home.tsx

```diff
@@ -23,28 +38,68 @@ export function Home() {
           <li>Insert other eligbility rules</li>
         </ul>
       </Banner>
-      <br />
-      <br />
     </>
   ) : null;
+
+  const handleSubmit = () => {
+    const postalCodeRegex = /[a-zA-Z][0-9][a-zA-Z](-| |)[0-9][a-zA-Z][0-9]/;
+    if (postalCodeRegex.test(postalCode)) {
+      setShouldRenderPharmacyList(true);
+      setShouldShowPostalPrompt(false);
+    } else {
+      setShouldShowInvalidPostal(true);
+    }
+  };
```

Comment:
Instead of setting a local state, do you think it's worthwhile to create a dedicated route for this? The problem is if the user refreshes, they'll get sent back to this postal code prompt.

Reply:
A route with the postal code as a route param would be nice so we could deep link a URL straight to the results screen.

There was also talk about using localStorage to store the postal code so users wouldn't have to re-enter it, so if we implement both of these flows, we would have they don't cross swords.